### ISSUE TITLE
Demonstration has no seed?

### ISSUE THREAD

**Issue by watchernyu** [facebookresearch/RRL#2]
Hi! I noticed that when converting the demonstrations, it will give this warning message.

`"++++++++++++++++++++++++++++ Couldn't find the seed of the demos. Please verify."`

When I print out the keys of the loaded demonstration file (using the commands below), it seems that the demonstrations don't really have the "seed" entry (I downloaded from the dapg repo), I'm not sure if this is the expected behavior? Thank you very much! 

```
import pickle
demo_paths = pickle.load(open('relocate-v0_demos.pickle', 'rb'))
print(len(demo_paths))
print(demo_paths[0].keys())

# output is: 
# 25
# dict_keys(['observations', 'init_state_dict', 'actions', 'rewards'])
```

**Comment by ShahRutav** [comment]
Hi @watchernyu. You are correct, the demonstrations provided by `hand_dapg` do not provide seed. However, I have used the same demonstrations, you can safely ignore the warning.

**Comment by watchernyu** [comment]
Thank you very much for the clarification! I'm closing this issue now.
